docs/security/trust/trust_automation.md

```diff
@@ -0,0 +1,77 @@
+<!--[metadata]>
++++
+title = "Automation with Trust"
+description = "Automating content push pulls with  trust"
+keywords = ["trust, security, docker,  documentation, automation"]
+[menu.main]
+parent= "smn_content_trust"
++++
+<![end-metadata]-->
+
+# Automation with Trust
+
+Your automation systems that pull or build images can also work with trust. Any automation environment must set `DOCKER_TRUST_ENABLED` either manually or in in a scripted fashion before processing images.
+
+## Bypass requests for passphrases
+
+To allow tools to wrap docker and push trusted content, there are three
+environment variables that allow you to provide the passphrases without an
+expect script, or typing them in:
+
+ - `DOCKER_CONTENT_TRUST_ROOT_PASSPHRASE`
+ - `DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE`
+
+Docker attempts to use the contents of these environment variables as passphrase
+for the keys. For example, an image creator can export the repository   `target`
+and `snapshot` passphrases:
+
+```bash
+$  export DOCKER_CONTENT_TRUST_ROOT_PASSPHRASE="u7pEQcGoebUHm6LHe6"
+$  export DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE="u7pEQcGoebUHm6LHe6"
+```
+
+Then, when pushing a new tag the Docker client does not request these values but signs automatically:
+
+``bash
+$  docker push diogomonica/alpine:latest
+The push refers to a repository [docker.io/diogomonica/alpine] (len: 1)
+a9539b34a6ab: Image already exists
+b3dbab3810fc: Image already exists
+latest: digest: sha256:d149ab53f871 size: 3355
+Signing and pushing trust metadata
+```
+
+## Building with trusted content
+
+You can also build with trust. Before running the `docker build` command, you should set the environment variable `DOCKER_TRUST_ENABLED` either manually or in in a scripted fashion. Consider the simple Dockerfile below. 
+
+```Dockerfile
+FROM diogomonica/alpine:latest
+RUN echo
+```
+
+The `FROM` tag is pulling a signed image. Given that trust data exists for the
+tag `latest`, the following build should succeed:
+
+```bash
+$  docker build -t diogomonica/alpine:testing .
+Using default tag: latest
+latest: Pulling from diogomonica/alpine
+
+b3dbab3810fc: Pull complete
+a9539b34a6ab: Pull complete
+Digest: sha256:d149ab53f871
+```
+
+If trust is enabled, building from a Dockerfile that relies on tag without trust data, causes the build command to fail:
```

Comment:
Maybe useful to show the different Dockerfile that uses `FROM diogomonica/alpine:notrust` first?